genai-on-vertex-ai/gemini/model_upgrades/instruction-following/promptfoo/README.md

```diff
@@ -0,0 +1,50 @@
+# RAG
+### _Eval Recipe for model migration_
+
+This Eval Recipe demonstrates how to compare performance of an Instruction Following prompt with Gemini 1.5 Flash and Gemini 2.0 Flash using an unlabeled dataset and open source evaluation tool [Promptfoo](https://www.promptfoo.dev/).
```

Comment:
Can we add a one sentence about what instruction following prompt means or link to the paper (https://arxiv.org/abs/2311.07911)? It's not as common as other use cases.